_data/sites.yml

```diff
@@ -1622,6 +1622,12 @@
   size: 3.0
   last_checked: 2022-04-26
 
+- domain: kaukokaipuu.xyz
+  url: https://www.kaukokaipuu
+  xyz/
```

Comment:
This still doesn't seem right. Do you mind fixing this?

Reply:
Now I think I got It right. Thanks for the patience. First timer with Github here.